infer_structcol/main.py

```diff
@@ -91,10 +87,8 @@ class Sample:
     incident_angle: scalar
         angle of incident light on the sample
     '''
-    def __init__(self, wavelength, particle_radius, thickness, particle_index, 
+    def __init__(self, wavelength, particle_index, 
```

Comment:
Maybe the ranges on phi, radius, and thickness should be included in the model, rather than when specified for an inference calculation? I'm not sure whether this should be the case, but it is worth considering...